pkg/controller/controller.go

```diff
@@ -490,12 +490,12 @@ func (ctrl *resizeController) markPVCAsFSResizeRequired(pvc *v1.PersistentVolume
 	_, err := ctrl.patchClaim(pvc, newPVC)
 
 	if err != nil {
-		return fmt.Errorf("Mark PVC %q as file system resize required failed: %v", util.PVCKey(pvc), err)
+		return fmt.Errorf("Failed to mark PVC %q for node expansion: %v", util.PVCKey(pvc), err)
 	}
 
-	klog.V(4).Infof("Mark PVC %q as file system resize required", util.PVCKey(pvc))
+	klog.V(4).Infof("Mark PVC %q for required node expansion", util.PVCKey(pvc))
 	ctrl.eventRecorder.Eventf(pvc, v1.EventTypeNormal,
-		util.FileSystemResizeRequired, "Require file system resize of volume on node")
+		util.FileSystemResizeRequired, "Require expansion of volume on node")
```

Comment:
I wonder if we can change the event keys though. The underlying motivation for this PR is - for raw block devices etc, they may not even need a file system expansion (but may still need rescannig the LUN on the node). So it is bit broad to call node expansion as a file system expansion.

Reply:
I renamed these keys. I think nobody should generally depend on the event keys.